src/docs/get-started/install/chromeos.md

```diff
@@ -31,62 +31,37 @@ must meet these minimum requirements:
   * `libGLU.so.1` - provided by mesa packages such as `libglu1-mesa` on
      Ubuntu/Debian
 
-For the best experience right now, you should put your
-Chrome OS Device into developer mode (this is necessary
-to push apps on the Chrome OS Device).  For more information,
-see [how to enable developer mode on your Chromebook][].
-
-{% include_relative _get-sdk.md %}
+{% include_relative _get-sdk-chromeos.md %}
 
 {% include_relative _path-linux-chromeos.md %}
 
-{% include_relative _chromeos-android-sdk-setup.md %}
+{% include_relative _android-setup-chromeos.md %}
 
 ## Next step
 
 Set up your preferred editor.
 
 ## Flutter & Chrome OS tips & tricks
 
-Wondering how to run your app? On Chrome OS,
-you can either connect your phone
-or push directly to the Android container on device.
-To do that you must enable Developer mode on your machine,
-and then connect to the local container with ADB:
-
-```terminal
-$ adb connect 100.115.92.2:5555
-```
-
-Want to build your first app optimized for Chrome OS?
-Clone the flutter-samples repo and build our specific Chrome
-OS Best Practices example:
-
-```terminal
-$ git clone https://github.com/flutter/samples
-$ cd samples/chrome-os-best-practices
-$ flutter run
-```
-
-Wondering how to access your favorite F-Key shortcuts
-on the Chrome OS keyboard?
-
-* Press the search key along with 1 through = to access F1–F12.
-
 For the current versions of Chrome OS, only certain ports from
 Crostini are exposed to the rest of the environments.
 Here’s an example of how to launch
 Flutter DevTools for an Android app with ports
 that will work:
 
 ```terminal
-$ flutter pub global run devtools -p 8000
+$ flutter packages pub global run devtools -p 8000
```

Comment:
When did this change? I thought we went from `flutter packages` to `flutter pub`.

Reply:
Thanks for the review, Brett! You're right. I reverted it back to `flutter pub`.

Reply:
Thanks for the PR, much appreciated!